akka-actor/src/main/scala/akka/actor/ActorCell.scala

```diff
@@ -408,7 +408,7 @@ private[akka] object ActorCell {
 private[akka] class ActorCell(
     val system: ActorSystemImpl,
     val self: InternalActorRef,
-    final val props: Props, // Must be final so that it can be properly cleared in clearActorCellFields
+    _initialProps: Props,
```

Comment:
`_initialProps` doesn't become an instance field since it's only used in constructor, right?

Reply:
That's the idea but haven't checked

Reply:
I confirmed that